HTML/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en" ng-app="testingApp">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Testing Project</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body ng-controller="MainController as mainCtrl">
+
+    <div class="level1">
+        <div class="level2">
+            <div class="level3">
+                <div class="level4">
+                    <div class="level5">
+                        Nested Divs up to 5 levels
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/17#issuecomment-2208262726>#007008</a><span> - 07/04/2024, 07:11 am</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Excessive nesting of div elements can lead to performance issues and make the HTML structure difficult to maintain. Deeply nested elements can also impact rendering performance and accessibility.
</div>
<div id="fix">
<b>Fix:</b> Reduce the level of nesting by flattening the structure or using CSS for layout purposes. Consider using semantic HTML elements where appropriate.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -10,12 +10,8 @@
 
      <div class="level1">
          <div class="level2">
 -            <div class="level3">
 -                <div class="level4">
 -                    <div class="level5">
 -                        Nested Divs up to 5 levels
 -                    </div>
 -                </div>
 -            </div>
 +                Nested Divs up to 2 levels
          </div>
      </div>
 
```
</div>
</details>
</div>
<br>


</div>